cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,298 @@
+        )
+
+    def handle(self, *args, **options):
+        super(Command, self).handle(*args, **options)
+        self.verbosity = int(options.get("verbosity", 1))
+        self.type = options.get("type")
+        self.id = options.get("id")
+
+        if not self.id:
+            self.stdout.write("Object id required!")
+            sys.exit(1)
+
+        if DEVELOPMENT:
+            if self.type == "Opinion":
+                self.clone_opinion()
+            elif self.type == "Docket":
+                self.clone_docket()
+            elif self.type == "Person":
+                self.clone_person()
+            elif self.type == "Court":
+                self.clone_court()
+            else:
+                self.stdout.write("Invalid type!")
+
+        else:
+            self.stdout.write("Command not enabled for production environment")
+
+    def clone_opinion(self) -> None:
+        """Download opinion cluster data from courtlistener.com and add it to
+        local environment
+        """
+
+        try:
+            obj = OpinionCluster.objects.get(pk=self.id)
+            print(f"OpinionCluster with id: {self.id} already in local env.")
+            return
+        except OpinionCluster.DoesNotExist:
+
+            s = requests.session()
+            s.headers = {
+                "Authorization": f"Token {os.environ.get('CL_API_TOKEN', '')}"
+            }
+            cluster_endpoint = f"https://www.courtlistener.com/api/rest/v3/clusters/{self.id}/"
+            results = s.get(cluster_endpoint).json()
+
+            cluster_datum = results
+            docket_datum = s.get(cluster_datum["docket"]).json()
+            court_datum = s.get(docket_datum["court"]).json()
+            citation_data = cluster_datum["citations"]
+            opinion_data = cluster_datum["sub_opinions"]
+            # delete resource_uri value generated by DRF
+            del court_datum["resource_uri"]
+            del docket_datum["resource_uri"]
+            del cluster_datum["resource_uri"]
+            # delete fields with fk or m2m relations or unneeded fields
+            del docket_datum["clusters"]
+            del docket_datum["original_court_info"]
+            del docket_datum["absolute_url"]
+            del docket_datum["audio_files"]
+            del docket_datum["tags"]
+            del docket_datum["panel"]
+            del cluster_datum["docket"]
+            del cluster_datum["citations"]
+            del cluster_datum["sub_opinions"]
+            del cluster_datum["absolute_url"]
+            del cluster_datum["panel"]
+            del cluster_datum["non_participating_judges"]
+            with transaction.atomic():
+                # Get or create required court
+                try:
+                    ct = Court.objects.get_or_create(**court_datum)
+                except:
+                    ct = Court.objects.filter(pk=court_datum["id"])
+                docket_datum["court"] = ct[0]
```

Comment:
This only works if we hit the exception and do the `filter`, right? Oh, wait, no, it works because `get_or_create` returns a two-tuple and `filter` returns a list. That's a little too magic. If you want to do that you should do:

```
try:
    ct, _ = ...get_or_create()
except SomeException:
    ct = ...filter()[0]
```